data/Templates/Hl7v2/Resource/_RelatedPerson.liquid

```diff
@@ -64,11 +64,11 @@
             {
                 {% include 'DataType/XTN' XTN: NK1.41 -%}
             },
-            { {% include 'DataType/XTN' XTN: PRT.15 -%} },
+            { {% include 'DataType/XTN' XTN: PRT.15.Repeats.0 -%} },
         ],
         "name":
         [
-            { {% include 'DataType/XPN' XPN: NK1.2 -%} },
+            { {% include 'DataType/XPN' XPN: NK1.2.Repeats.0 -%} },
```

Comment:
Seen from the second commit of functional tests: We should use (a) `NK1.2.Repeats[0]` instead of (b) `NK1.2.Repeats.0` here. "`.`" operator always goes into a deeper level, e.g., from segment to field, from field to component, from component to subcomponent, so (b) refers to component NK1.2.0, which is always null (component starts from index 1).

Reply:
Sure, I'll recover it.
The `.Repeats[0]` will cause crash while converting, and the `Repeats.0` works. I'll investigate the reason and try to fix it.

Reply:
After changing from `.Repeats[0]` to `Repeats[0].Value`, the crashing disappeared. I'll update the repeatable identifiers in the main template fixing branch.

Reply:
Yes, `Repeats[0]` is a Hl7v2Field object and `Repeats[0].Value` is a string. Which one to use depends on the scenario: When used in a `include` snippet, we usually pass a Hl7v2Field. If we want to render it directly, we usually use `Repeats[0].Value`.